Article/Article.js

```diff
@@ -3,17 +3,24 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'EXPAND';
     // Set a click handler on the expandButton reference, calling the expandArticle method.
     
+    this.expandButton.addEventListener('click', () =>{
+      this.expandArticle();
+    });
+
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
+    this.domElement.classList.toggle('article-open');
```

Comment:
Very smart to use classList, it gives you a lot of convenient options to manipulate classes!